qtt/instrument_drivers/virtualAwg/templates.py

```diff
@@ -60,3 +60,19 @@ def marker(name):
         """
         return TablePT({name: [(0, 0), ('offset', 1),
                                ('offset+uptime', 0), ('period', 0)]})
+
+    @staticmethod
+    def rollover_marker(name):
+        """Creates a TTL pulse qupulse template for sequencing.
```

Comment:
I understand the parameters of this pulse template, because I just read the (private) `__calculate_pulse_parameters`. Without that I would have a hard time understanding what kind of pulse this is. The usage of the boolean `rollover` variable is particularly "interesting".

The parameters for this pulse are generated from `__calculate_pulse_parameters` which is ok, but if this was a pulse template to be used by a user, I would like a different interface (for example only the parameters `pulse_start`, `pulse_length` and `period` and let the template do calculations internally)